bitnami/mariadb-galera/README.md

```diff
@@ -350,6 +352,95 @@ extraContainers:
       memory: 10Mi
 ```
 
+### Bootstraping a node other than 0
+
+> Note: Some of these procedures can lead to data loss, always make a backup beforehand.
+
+To restart the cluster you need to check state in which it is after being stopped, also you will need the previous password for the `rootUser` and `mariabackup`, and the deployment name. The value of `safe_to_bootstrap` in `/bitnami/mariadb/data/grastate.dat`, will indicate if it safe to bootstrap form that node. In the case it is other than node 0, it is needed to choose one and force the bootstraping from it.
```

Comment:
`to check state` -> `to check the state`
`if it safe` -> `if it is safe`
From my point the comma at `in `/bitnami/mariadb/data/grastate.dat`, will` is not needed.